delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/model/acls/Target.scala

```diff
@@ -29,32 +22,46 @@ sealed trait Target extends Product with Serializable {
 
 object Target {
 
+  /**
+    * An ACL target location where a certain ACL is anchored.
+    */
+  sealed trait TargetLocation extends Target {
```

Comment:
I think that possibly a disjoint ADT would be better, something like:
```
sealed trait Target
case object Root
case class Org(label)
case class Proj(label, label)
```
and
```
sealed trait TargetSelection
case object AnyOrg  /*
case object AnyOrgAnyProj  /*/*
case class FixedOrgAnyProj(label) /label/*
```
... because it allows one to discriminate between fixed addressable resources (targets) and queries (target selections). I don't think there's a need to unify these as it generates confusion. The routes will be able to identify for GET requests if the provided segments are wildcards or fixed values and choose the appropriate intent.

Reply:
It is easier for me as I did it. In listings you can pass either one or another, so there is a reason to have a super trait for both.

Reply:
Target is a superclass for `TargetLocation` and `TargetFilterLocation`, so you can select whatever you want :)